rpc/server.go

```diff
@@ -146,6 +151,15 @@ type simpleServer struct {
 
 	// authIssuer is the JWT issuer (iss) that will be used for our service.
 	authIssuer string
+
+	counters struct {
+		TcpGrpcRequestsStarted      atomic.Int64
```

Comment:
Obviously lint on all of these, but probably also some comments describing what they're counting (I realize it's pretty obvious.)

Reply:
Sure; sounds good to me. Thanks for the reasoning.